### PR TITLE
Transformations: Move moving average and cumulative sum to private preview

### PR DESCRIPTION
**Pull request by oscarkilhed**
**What is this feature?**

This sets the moving average and cumulative sum feature flag to private preview.

### CODE REVIEW

**Review by imatwawana — APPROVED** [review]

**Review by mdvictor — APPROVED** [review]

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 1]
```diff
@@ -1307,4 +1307,4 @@ There are two different models:
 
 {{% /docs/reference %}}
 
-[Data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
+[data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
```
Comment (baldm0mma) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1310]: I think this happened last time as well, tho I can't remember why. But this should not be changed to lowercase.

Comment (baldm0mma) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1310]:
```suggestion
[Data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
```

**Review by baldm0mma — CHANGES_REQUESTED** [review]
Small case change.

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md (hunk 1 above)
Comment (oscarkilhed) [reply docs/sources/panels-visualizations/query-transform-data/transform-data/index.md]: yes, seems like a bug with the generation of the help file

**Review by baldm0mma — APPROVED** [review]
Great!

**Pull request merged**